### PR TITLE
DOC-1875: RC - Billing alert user limit

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft

### CODE REVIEW

**Comment by lanceleonard** [comment]
A staged preview [is available](https://docs.redis.com/staging/jira-doc-1875/rc/security/access-management/).

File: content/rc/security/access-management.md [hunk 1]
```diff
@@ -61,8 +61,8 @@ Use this dialog to specify the following values:
 | **Email** | The address used for alerts and other email messages regarding the account | 
 | **Alert emails** | Enable to be notified when subscription databases cross certain thresholds, such as exceeding memory limits or latency requirements |
 | **Operational emails** | Notifications about subscription and database changes, such as creating or deleting a database |
-| **Billing emails** | Notifications when bills are issued, paid, and so on |
-| **Multi-factor authentication** | Whether MFA is enabled for the member.  This is deactivated when members have not enabled or confirmed MFA in their user profile settings |
+| **Billing emails** | Notifications when bills are issued, paid, and so on.<br/>_Note:_ Limited to a single user. |
```
Comment (cmilesb) [inline content/rc/security/access-management.md:64]:
```suggestion
| **Billing emails** | Notifications when bills are issued or paid.<br/>_Note:_ Limited to a single user. |
```

**Review by cmilesb — APPROVED** [review]
Looks good to me. Just suggested one wording change.

File: content/rc/security/access-management.md (hunk 1 above)
Comment (lanceleonard) [reply content/rc/security/access-management.md:64]: Fixed

**Pull request merged**